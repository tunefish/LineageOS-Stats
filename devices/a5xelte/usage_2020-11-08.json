{
  "0.13-q-2020102781252-test-a5xelte": 1,
  "15.1-20190426-unofficial-a5xelte": 1,
  "15.1-20190517-unofficial-a5xelte": 2,
  "15.1-20190526-unofficial-a5xelte": 4,
  "15.1-20190623-unofficial-a5xelte": 2,
  "15.1-20190709-unofficial-a5xelte": 12,
  "16.0-20190121-unofficial-a5xelte": 11,
  "16.0-20190625-unofficial-a5xelte": 6,
  "16.0-20190627-unofficial-a5xelte": 1,
  "16.0-20190707-unofficial-a5xelte": 8,
  "16.0-20190723-unofficial-a5xelte": 6,
  "16.0-20190811-unofficial-a5xelte": 1,
  "16.0-20190816-unofficial-a5xelte": 2,
  "16.0-20190818-unofficial-a5xelte": 21,
  "16.0-20190824-nightly-a5xelte": 1,
  "16.0-20190827-nightly-a5xelte": 1,
  "16.0-20190903-nightly-a5xelte": 1,
  "16.0-20190904-nightly-a5xelte": 2,
  "16.0-20190905-nightly-a5xelte": 1,
  "16.0-20190908-nightly-a5xelte": 1,
  "16.0-20190909-unofficial-a5xelte": 1,
  "16.0-20190913-nightly-a5xelte": 4,
  "16.0-20190914-nightly-a5xelte": 1,
  "16.0-20190916-nightly-a5xelte": 1,
  "16.0-20190917-nightly-a5xelte": 2,
  "16.0-20190921-nightly-a5xelte": 2,
  "16.0-20190925-nightly-a5xelte": 1,
  "16.0-20190926-nightly-a5xelte": 1,
  "16.0-20190930-nightly-a5xelte": 1,
  "16.0-20191002-nightly-a5xelte": 1,
  "16.0-20191009-nightly-a5xelte": 1,
  "16.0-20191013-nightly-a5xelte": 2,
  "16.0-20191016-nightly-a5xelte": 1,
  "16.0-20191018-nightly-a5xelte": 1,
  "16.0-20191022-nightly-a5xelte": 3,
  "16.0-20191026-nightly-a5xelte": 1,
  "16.0-20191029-nightly-a5xelte": 2,
  "16.0-20191030-nightly-a5xelte": 1,
  "16.0-20191101-nightly-a5xelte": 2,
  "16.0-20191104-nightly-a5xelte": 1,
  "16.0-20191105-nightly-a5xelte": 3,
  "16.0-20191107-nightly-a5xelte": 1,
  "16.0-20191109-nightly-a5xelte": 1,
  "16.0-20191110-nightly-a5xelte": 4,
  "16.0-20191112-nightly-a5xelte": 3,
  "16.0-20191114-nightly-a5xelte": 1,
  "16.0-20191116-nightly-a5xelte": 1,
  "16.0-20191121-nightly-a5xelte": 2,
  "16.0-20191127-nightly-a5xelte": 1,
  "16.0-20191130-nightly-a5xelte": 2,
  "16.0-20191201-nightly-a5xelte": 1,
  "16.0-20191205-nightly-a5xelte": 7,
  "16.0-20191209-nightly-a5xelte": 2,
  "16.0-20191210-nightly-a5xelte": 3,
  "16.0-20191211-nightly-a5xelte": 2,
  "16.0-20191214-nightly-a5xelte": 1,
  "16.0-20191215-nightly-a5xelte": 1,
  "16.0-20191219-nightly-a5xelte": 1,
  "16.0-20191222-nightly-a5xelte": 3,
  "16.0-20191224-nightly-a5xelte": 1,
  "16.0-20191227-nightly-a5xelte": 3,
  "16.0-20191228-nightly-a5xelte": 1,
  "16.0-20191229-nightly-a5xelte": 2,
  "16.0-20191230-nightly-a5xelte": 1,
  "16.0-20191231-nightly-a5xelte": 3,
  "16.0-20200101-nightly-a5xelte": 1,
  "16.0-20200103-nightly-a5xelte": 2,
  "16.0-20200106-nightly-a5xelte": 3,
  "16.0-20200109-nightly-a5xelte": 1,
  "16.0-20200111-nightly-a5xelte": 2,
  "16.0-20200113-nightly-a5xelte": 2,
  "16.0-20200115-nightly-a5xelte": 2,
  "16.0-20200116-nightly-a5xelte": 2,
  "16.0-20200117-nightly-a5xelte": 1,
  "16.0-20200118-nightly-a5xelte": 2,
  "16.0-20200119-nightly-a5xelte": 1,
  "16.0-20200123-nightly-a5xelte": 1,
  "16.0-20200124-nightly-a5xelte": 2,
  "16.0-20200130-nightly-a5xelte": 1,
  "16.0-20200203-microg-a5xelte": 2,
  "16.0-20200203-nightly-a5xelte": 1,
  "16.0-20200204-nightly-a5xelte": 2,
  "16.0-20200205-nightly-a5xelte": 2,
  "16.0-20200207-nightly-a5xelte": 2,
  "16.0-20200208-nightly-a5xelte": 2,
  "16.0-20200209-nightly-a5xelte": 1,
  "16.0-20200212-nightly-a5xelte": 1,
  "16.0-20200213-nightly-a5xelte": 1,
  "16.0-20200215-nightly-a5xelte": 1,
  "16.0-20200217-nightly-a5xelte": 1,
  "16.0-20200218-nightly-a5xelte": 1,
  "16.0-20200220-nightly-a5xelte": 2,
  "16.0-20200221-nightly-a5xelte": 3,
  "16.0-20200222-nightly-a5xelte": 3,
  "16.0-20200224-nightly-a5xelte": 7,
  "16.0-20200229-nightly-a5xelte": 2,
  "16.0-20200301-nightly-a5xelte": 1,
  "16.0-20200303-nightly-a5xelte": 1,
  "16.0-20200306-nightly-a5xelte": 1,
  "16.0-20200307-nightly-a5xelte": 2,
  "16.0-20200308-nightly-a5xelte": 2,
  "16.0-20200309-nightly-a5xelte": 3,
  "16.0-20200310-nightly-a5xelte": 3,
  "16.0-20200311-nightly-a5xelte": 6,
  "16.0-20200312-nightly-a5xelte": 1,
  "16.0-20200313-nightly-a5xelte": 7,
  "16.0-20200314-nightly-a5xelte": 2,
  "16.0-20200315-nightly-a5xelte": 1,
  "16.0-20200317-nightly-a5xelte": 5,
  "16.0-20200319-nightly-a5xelte": 2,
  "16.0-20200320-nightly-a5xelte": 2,
  "16.0-20200321-nightly-a5xelte": 1,
  "16.0-20200322-nightly-a5xelte": 2,
  "16.0-20200323-microg-a5xelte": 1,
  "16.0-20200324-nightly-a5xelte": 7,
  "16.0-20200325-nightly-a5xelte": 28,
  "16.0-20200331-microg-a5xelte": 1,
  "16.0-20200412-nightly-a5xelte": 11,
  "16.0-20200419-nightly-a5xelte": 176,
  "16.0-20200422-microg-a5xelte": 1,
  "16.0-20200427-nightly-a5xelte": 18,
  "16.0-20200502-microg-a5xelte": 1,
  "16.0-20200507-microg-a5xelte": 1,
  "16.0-20200521-microg-a5xelte": 2,
  "17.0-20200108-unofficial-a5xelte": 1,
  "17.1-20200117-unofficial-a5xelte": 9,
  "17.1-20200121-unofficial-a5xelte": 8,
  "17.1-20200207-unofficial-a5xelte": 1,
  "17.1-20200210-unofficial-a5xelte": 3,
  "17.1-20200216-unofficial-a5xelte": 1,
  "17.1-20200222-unofficial-a5xelte": 1,
  "17.1-20200229-unofficial-a5xelte": 2,
  "17.1-20200307-unofficial-a5xelte": 2,
  "17.1-20200312-unofficial-a5xelte": 10,
  "17.1-20200412-unofficial-a5xelte": 15,
  "17.1-20200516-unofficial-a5xelte": 118,
  "17.1-20200529-nightly-a5xelte": 1,
  "17.1-20200530-nightly-a5xelte": 4,
  "17.1-20200601-nightly-a5xelte": 7,
  "17.1-20200603-nightly-a5xelte": 1,
  "17.1-20200604-nightly-a5xelte": 5,
  "17.1-20200606-unofficial-a5xelte": 1,
  "17.1-20200607-nightly-a5xelte": 11,
  "17.1-20200614-nightly-a5xelte": 8,
  "17.1-20200621-nightly-a5xelte": 7,
  "17.1-20200628-nightly-a5xelte": 29,
  "17.1-20200712-nightly-a5xelte": 21,
  "17.1-20200715-unofficial-tissot": 11,
  "17.1-20200716-unofficial-herolte": 10,
  "17.1-20200719-nightly-a5xelte": 19,
  "17.1-20200726-nightly-a5xelte": 45,
  "17.1-20200809-nightly-a5xelte": 69,
  "17.1-20200818-microg-a5xelte": 8,
  "17.1-20200821-unofficial-z3": 56,
  "17.1-20200824-microg-a5xelte": 2,
  "17.1-20200830-nightly-a5xelte": 42,
  "17.1-20200904-unofficial-a5xelte": 1,
  "17.1-20200906-nightly-a5xelte": 34,
  "17.1-20200913-nightly-a5xelte": 70,
  "17.1-20200918-unofficial-a5xelte": 1,
  "17.1-20200920-nightly-a5xelte": 235,
  "17.1-20200921-microg-a5xelte": 9,
  "17.1-20200927-nightly-a5xelte": 219,
  "17.1-20201004-nightly-a5xelte": 182,
  "17.1-20201011-nightly-a5xelte": 201,
  "17.1-20201012-microg-a5xelte": 9,
  "17.1-20201014-unofficial-a5xelte": 2,
  "17.1-20201016-unofficial-a5xelte": 1,
  "17.1-20201017-unofficial-a5xelte": 2,
  "17.1-20201018-nightly-a5xelte": 154,
  "17.1-20201023-unofficial-a5xelte": 1,
  "17.1-20201025-nightly-a5xelte": 214,
  "17.1-20201027-unofficial-a5xelte": 1,
  "17.1-20201101-nightly-a5xelte": 322,
  "17.1-20201105-microg-a5xelte": 2,
  "18.0-20200927-unofficial-a5xelte": 1,
  "18.0-20201013-unofficial-a5xelte": 2,
  "18.0-20201022-unofficial-a5xelte": 3
}