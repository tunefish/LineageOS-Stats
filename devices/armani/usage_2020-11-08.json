{
  "13.0-20170113-unofficial-armani": 16,
  "13.0-20170224-unofficial-armani": 11,
  "13.0-20170713-unofficial-armani": 6,
  "14.1-20170123-nightly-armani": 1,
  "14.1-20170130-nightly-armani": 1,
  "14.1-20170206-nightly-armani": 2,
  "14.1-20170212-nightly-armani": 8,
  "14.1-20170313-nightly-armani": 3,
  "14.1-20170320-nightly-armani": 3,
  "14.1-20170403-nightly-armani": 1627,
  "14.1-20170410-nightly-armani": 4,
  "14.1-20170424-nightly-armani": 1,
  "14.1-20170508-nightly-armani": 5,
  "14.1-20170515-nightly-armani": 3,
  "14.1-20170522-nightly-armani": 2,
  "14.1-20170529-nightly-armani": 1,
  "14.1-20170605-nightly-armani": 6,
  "14.1-20170612-nightly-armani": 1,
  "14.1-20170619-nightly-armani": 2,
  "14.1-20170622-unofficial-armani": 1,
  "14.1-20170626-nightly-armani": 8,
  "14.1-20170703-nightly-armani": 4,
  "14.1-20170710-nightly-armani": 1,
  "14.1-20170717-nightly-armani": 3,
  "14.1-20170724-nightly-armani": 3,
  "14.1-20170807-nightly-armani": 1,
  "14.1-20170811-unofficial-armani": 1,
  "14.1-20170814-nightly-armani": 4,
  "14.1-20170821-nightly-armani": 1,
  "14.1-20170828-nightly-armani": 1,
  "14.1-20170904-nightly-armani": 3,
  "14.1-20170911-nightly-armani": 2,
  "14.1-20170918-nightly-armani": 1,
  "14.1-20170925-nightly-armani": 2,
  "14.1-20171002-nightly-armani": 5,
  "14.1-20171009-nightly-armani": 2,
  "14.1-20171016-nightly-armani": 3,
  "14.1-20171023-nightly-armani": 2,
  "14.1-20171030-nightly-armani": 1,
  "14.1-20171106-nightly-armani": 4,
  "14.1-20171113-nightly-armani": 3,
  "14.1-20171120-nightly-armani": 5,
  "14.1-20171204-nightly-armani": 3,
  "14.1-20171211-nightly-armani": 1,
  "14.1-20171218-nightly-armani": 2,
  "14.1-20171225-nightly-armani": 2,
  "14.1-20180101-nightly-armani": 4,
  "14.1-20180108-nightly-armani": 1,
  "14.1-20180108-unofficial-armani": 1,
  "14.1-20180116-unofficial-armani": 1,
  "14.1-20180122-nightly-armani": 3,
  "14.1-20180129-nightly-armani": 2,
  "14.1-20180130_084918-unofficial-armani": 1,
  "14.1-20180205-nightly-armani": 4,
  "14.1-20180212-nightly-armani": 3,
  "14.1-20180215-unofficial-armani": 4,
  "14.1-20180219-nightly-armani": 10,
  "14.1-20180228-nightly-armani": 2,
  "14.1-20180307-nightly-armani": 1,
  "14.1-20180314-nightly-armani": 4,
  "14.1-20180314-unofficial-armani": 1,
  "14.1-20180321-nightly-armani": 2,
  "14.1-20180328-nightly-armani": 6,
  "14.1-20180404-nightly-armani": 1,
  "14.1-20180410-nightly-armani": 2,
  "14.1-20180418-nightly-armani": 6,
  "14.1-20180425-nightly-armani": 2,
  "14.1-20180503-nightly-armani": 6,
  "14.1-20180509-nightly-armani": 3,
  "14.1-20180516-nightly-armani": 4,
  "14.1-20180523-nightly-armani": 27,
  "14.1-20180530-nightly-armani": 5,
  "14.1-20180606-nightly-armani": 8,
  "14.1-20180613-nightly-armani": 3,
  "14.1-20180620-nightly-armani": 2,
  "14.1-20180627-nightly-armani": 7,
  "14.1-20180704-nightly-armani": 7,
  "14.1-20180711-nightly-armani": 4,
  "14.1-20180718-nightly-armani": 7,
  "14.1-20180725-nightly-armani": 2,
  "14.1-20180801-nightly-armani": 8,
  "14.1-20180808-nightly-armani": 5,
  "14.1-20180815-nightly-armani": 1,
  "14.1-20180822-nightly-armani": 8,
  "14.1-20180829-nightly-armani": 3,
  "14.1-20180905-nightly-armani": 3,
  "14.1-20180912-nightly-armani": 6,
  "14.1-20180926-nightly-armani": 5,
  "14.1-20181003-nightly-armani": 3,
  "14.1-20181010-nightly-armani": 8,
  "14.1-20181017-nightly-armani": 11,
  "14.1-20181024-nightly-armani": 10,
  "14.1-20181031-nightly-armani": 14,
  "14.1-20181107-microg-armani": 2,
  "14.1-20181107-nightly-armani": 154,
  "14.1-20181121-microg-armani": 1,
  "14.1-20190717-unofficial-armani": 14,
  "14.1-20190819-unofficial-armani": 72,
  "14.1-20200616-unofficial-armani": 3,
  "14.1-20201006-unofficial-armani": 48,
  "14.1-20201029-unofficial-armani": 1,
  "15.1-20180209-unofficial-armani": 79,
  "16.0-20190218-unofficial-armani": 4,
  "aim_armani-userdebug 7.1.2 n2g48h c24542215b test-keys": 3,
  "cm_armani-userdebug 5.1.1 lvy48f 7da1783e95 test-keys": 1,
  "jls36c": 1,
  "n2g48h test-keys": 10,
  "njh47d": 5,
  "to_armani-userdebug 7.1.2 njh47f 491dadf9f3 test-keys": 2
}