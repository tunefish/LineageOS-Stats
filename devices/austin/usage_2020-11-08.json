{
  "12.1-20181218-unofficial-austin": 1211,
  "12.1-20190418-unofficial-austin": 8,
  "12.1-20190422-unofficial-austin": 21,
  "12.1-20190524-unofficial-austin": 4,
  "12.1-20190731-unofficial-austin": 4,
  "12.1-20190823-unofficial-austin": 5,
  "12.1-20200423-unofficial-austin": 2,
  "12.1-20200505-unofficial-austin": 35,
  "14.1-20190828-unofficial-austin": 268,
  "4.8-20190418-wechat-fwrenren": 1
}