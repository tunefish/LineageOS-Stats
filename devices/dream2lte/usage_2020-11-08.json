{
  "14.1-20171005-unofficial-dream2lte": 8,
  "14.1-20171120_192719-unofficial-dream2lte": 1,
  "15.1-20190304-unofficial-dream2lte": 1,
  "16.0-20190325-unofficial-dream2lte": 14,
  "16.0-20190406-unofficial-dream2lte": 2,
  "16.0-20190527-unofficial-dream2lte": 1,
  "16.0-20190618-unofficial-dream2lte": 3,
  "16.0-20190829-unofficial-dream2lte": 6,
  "16.0-20190830-unofficial-dream2lte": 4,
  "16.0-20190922-unofficial-dream2lte": 25,
  "16.0-20191102-unofficial-dream2lte": 40,
  "17.0-20191222-unofficial-dream2lte": 4,
  "17.1-20200104-unofficial-dream2lte": 3,
  "17.1-20200108-unofficial-dream2lte": 21,
  "17.1-20200112-unofficial-dream2lte": 30,
  "17.1-20200128-unofficial-dream2lte": 36,
  "17.1-20200131-unofficial-dream2lte": 8,
  "17.1-20200209-unofficial-dream2lte": 4,
  "17.1-20200211-unofficial-dream2lte": 26,
  "17.1-20200411-unofficial-dream2lte": 12,
  "17.1-20200415-unofficial-dream2lte": 596,
  "17.1-20200506-unofficial-dream2lte": 5,
  "17.1-20200515-unofficial-dream2lte": 6,
  "17.1-20200615-unofficial-dream2lte": 5,
  "17.1-20200715-unofficial-dream2lte": 11,
  "17.1-20200715-unofficial-tissot": 9,
  "17.1-20200716-unofficial-herolte": 5,
  "17.1-20200815-unofficial-dream2lte": 27,
  "17.1-20200821-unofficial-z3": 3784,
  "17.1-20200915-unofficial-dream2lte": 138,
  "17.1-20200924-unofficial-dream2lte": 2,
  "17.1-20201014-unofficial-herolte": 1,
  "17.1-20201015-unofficial-dream2lte": 219,
  "17.1-20201015-unofficial-starlte": 1,
  "17.1-20201102-unofficial-dream2lte": 1,
  "5.1-20200927-sana_public-dream2lte": 1,
  "qq2a.200501.001.b2": 11,
  "qq3a.200605.001": 3,
  "qq3a.200705.002": 5,
  "qq3a.200805.001": 358
}