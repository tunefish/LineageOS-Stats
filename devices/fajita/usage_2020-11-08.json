{
  "0.11-q-2020090772203-dev-fajita": 21,
  "0.12-q-2020103081675-dev-fajita": 3,
  "0.12-q-2020110282350-dev-fajita": 4,
  "0.12-q-2020110482855-dev-fajita": 2,
  "10.0-20201018-unofficial-snapshot-fajita": 2,
  "10.0-20201019-unofficial-snapshot-fajita": 10,
  "10.0-20201023-unofficial-snapshot-fajita": 2,
  "10.0-20201024-unofficial-snapshot-fajita": 2,
  "16.0-20190206_155036-unofficial-fajita": 1,
  "16.0-20190304_202959-unofficial-fajita": 1,
  "16.0-20190320_235022-unofficial-fajita": 1,
  "16.0-20190410_142527-unofficial-fajita": 1,
  "16.0-20190510_200016-unofficial-fajita": 2,
  "16.0-20190608_121231-unofficial-fajita": 1,
  "16.0-20190704_122426-unofficial-fajita": 1,
  "16.0-20190706_180032-unofficial-fajita": 1,
  "16.0-20190710_084738-unofficial-fajita": 1,
  "16.0-20190713_151545-unofficial-fajita": 1,
  "16.0-20190718_095751-unofficial-fajita": 4,
  "16.0-20190802_103124-unofficial-fajita": 2,
  "16.0-20190811_110733-unofficial-fajita": 5,
  "16.0-20190825-nightly-fajita": 1,
  "16.0-20190829-nightly-fajita": 1,
  "16.0-20190905-nightly-fajita": 1,
  "16.0-20190908-nightly-fajita": 1,
  "16.0-20190910-nightly-fajita": 1,
  "16.0-20190921-nightly-fajita": 2,
  "16.0-20190924-nightly-fajita": 1,
  "16.0-20190927-nightly-fajita": 1,
  "16.0-20191006-nightly-fajita": 1,
  "16.0-20191010-nightly-fajita": 7,
  "16.0-20191012-nightly-fajita": 12,
  "16.0-20191014-nightly-fajita": 20,
  "16.0-20191017-unofficial-fajita": 2,
  "16.0-20191022-nightly-fajita": 3,
  "16.0-20191023-nightly-fajita": 1,
  "16.0-20191028-nightly-fajita": 1,
  "16.0-20191106-nightly-fajita": 1,
  "16.0-20191110-nightly-fajita": 1,
  "16.0-20191123-nightly-fajita": 2,
  "16.0-20191126-nightly-fajita": 1,
  "16.0-20191129-nightly-fajita": 1,
  "16.0-20191208-nightly-fajita": 1,
  "16.0-20191209-nightly-fajita": 1,
  "16.0-20191216-nightly-fajita": 2,
  "16.0-20191218-nightly-fajita": 1,
  "16.0-20191224-nightly-fajita": 1,
  "16.0-20191231-nightly-fajita": 1,
  "16.0-20200107-nightly-fajita": 1,
  "16.0-20200113-nightly-fajita": 1,
  "16.0-20200114-nightly-fajita": 1,
  "16.0-20200115-nightly-fajita": 2,
  "16.0-20200116-nightly-fajita": 1,
  "16.0-20200123-nightly-fajita": 3,
  "16.0-20200131-nightly-fajita": 1,
  "16.0-20200202-nightly-fajita": 2,
  "16.0-20200203-nightly-fajita": 1,
  "16.0-20200208-nightly-fajita": 1,
  "16.0-20200212-nightly-fajita": 1,
  "16.0-20200215-nightly-fajita": 2,
  "16.0-20200217-nightly-fajita": 1,
  "16.0-20200220-nightly-fajita": 1,
  "16.0-20200223-nightly-fajita": 1,
  "16.0-20200301-nightly-fajita": 1,
  "16.0-20200304-nightly-fajita": 1,
  "16.0-20200308-nightly-fajita": 4,
  "16.0-20200309-nightly-fajita": 17,
  "16.0-20200312-nightly-fajita": 2,
  "16.0-20200314-nightly-fajita": 1,
  "16.0-20200318-nightly-fajita": 3,
  "16.0-20200319-nightly-fajita": 2,
  "16.0-20200322-nightly-fajita": 1,
  "16.0-20200325-nightly-fajita": 80,
  "16.0-20200326-nightly-fajita": 2,
  "16.0-20201021-unofficial-fajita": 1,
  "17.0-20190921_100536-unofficial-fajita": 3,
  "17.0-20191206_140358-unofficial-fajita": 1,
  "17.1-20200115_153235-unofficial-fajita": 2,
  "17.1-20200130_125605-unofficial-fajita": 3,
  "17.1-20200223_133843-unofficial-fajita": 1,
  "17.1-20200318_120849-unofficial-fajita": 2,
  "17.1-20200325_125535-unofficial-fajita": 1,
  "17.1-20200329-unofficial-fajita": 1,
  "17.1-20200403-nightly-fajita": 1,
  "17.1-20200405-nightly-fajita": 1,
  "17.1-20200406-nightly-fajita": 1,
  "17.1-20200409-nightly-fajita": 1,
  "17.1-20200410-nightly-fajita": 1,
  "17.1-20200415-nightly-fajita": 1,
  "17.1-20200421-nightly-fajita": 2,
  "17.1-20200422-nightly-fajita": 4,
  "17.1-20200425-nightly-fajita": 1,
  "17.1-20200427-nightly-fajita": 1,
  "17.1-20200529-nightly-fajita": 1,
  "17.1-20200602-nightly-fajita": 18,
  "17.1-20200605-nightly-fajita": 8,
  "17.1-20200611-nightly-fajita": 5,
  "17.1-20200618-nightly-fajita": 5,
  "17.1-20200625-nightly-fajita": 4,
  "17.1-20200702-nightly-fajita": 7,
  "17.1-20200709-nightly-fajita": 8,
  "17.1-20200716-nightly-fajita": 4,
  "17.1-20200723-nightly-fajita": 7,
  "17.1-20200730-nightly-fajita": 5,
  "17.1-20200806-nightly-fajita": 14,
  "17.1-20200813-nightly-fajita": 27,
  "17.1-20200817-unofficial-fajita": 1,
  "17.1-20200819-microg-fajita": 4,
  "17.1-20200825-microg-fajita": 9,
  "17.1-20200827-nightly-fajita": 13,
  "17.1-20200903-nightly-fajita": 23,
  "17.1-20200905-unofficial-fajita": 1,
  "17.1-20200910-nightly-fajita": 30,
  "17.1-20200917-nightly-fajita": 65,
  "17.1-20200922-microg-fajita": 10,
  "17.1-20200924-nightly-fajita": 133,
  "17.1-20201001-nightly-fajita": 139,
  "17.1-20201008-nightly-fajita": 157,
  "17.1-20201014-microg-fajita": 47,
  "17.1-20201014-unofficial-fajita": 2,
  "17.1-20201015-nightly-fajita": 170,
  "17.1-20201022-nightly-fajita": 304,
  "17.1-20201031-unofficial-fajita": 1,
  "17.1-20201101_091248-unofficial-fajita": 1,
  "17.1-20201105-nightly-fajita": 91,
  "17.1-20201105-unofficial-fajita": 1,
  "18.0-20201009_230604-unofficial-fajita": 79,
  "18.0-20201010-unofficial-fajita": 1,
  "18.0-20201011-unofficial-fajita": 1,
  "18.0-20201011_152913-unofficial-fajita": 26,
  "18.0-20201013_193454-unofficial-fajita": 4,
  "18.0-20201014-unofficial-fajita": 1,
  "18.0-20201014_141307-unofficial-fajita": 16,
  "18.0-20201016_030747-unofficial-fajita": 10,
  "18.0-20201017_063916-unofficial-fajita": 4,
  "18.0-20201018_043009-unofficial-fajita": 13,
  "18.0-20201019_210458-unofficial-fajita": 13,
  "18.0-20201021_212020-unofficial-fajita": 22,
  "18.0-20201024_152712-unofficial-fajita": 16,
  "18.0-20201027_154748-unofficial-fajita": 25,
  "18.0-20201028-unofficial-fajita": 2,
  "18.0-20201029-unofficial-fajita": 1,
  "18.0-20201031-unofficial-fajita": 2,
  "18.0-20201031_005757-unofficial-fajita": 57,
  "18.0-20201101-unofficial-fajita": 2,
  "18.0-20201103-unofficial-fajita": 1,
  "18.0-20201104-unofficial-fajita": 2,
  "18.0-20201107-unofficial-fajita": 1,
  "18.0-20201107_022607-unofficial-fajita": 4,
  "5.1-20200824-sana_public-fajita": 1,
  "rr_fajita-userdebug 10 qq3a.200805.001 eng.anieri.20200816.035721 test-keys": 2
}