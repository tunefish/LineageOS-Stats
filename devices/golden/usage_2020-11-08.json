{
  "13.0-20170118-unofficial-golden": 1,
  "13.0-20170121-unofficial-golden": 1,
  "13.0-20170129-unofficial-golden": 67,
  "13.0-20170224-unofficial-golden": 5,
  "13.0-20170302-unofficial-golden": 5,
  "13.0-20170310-unofficial-golden": 11,
  "13.0-20170328-unofficial-golden": 372,
  "13.0-20170420-unofficial-golden": 27,
  "14.1-20170122-unofficial-golden": 172,
  "14.1-20170318-unofficial-golden": 4,
  "14.1-20170402-unofficial-golden": 3,
  "14.1-20170415-unofficial-golden": 5,
  "14.1-20170419-unofficial-golden": 4,
  "14.1-20170507-unofficial-golden": 15,
  "14.1-20170522-unofficial-golden": 1,
  "14.1-20170609-unofficial-golden": 306,
  "14.1-20170610-unofficial-golden": 6,
  "14.1-20170628-unofficial-golden": 531,
  "14.1-20170724-unofficial-golden": 29,
  "14.1-20170817-unofficial-golden": 19,
  "14.1-20170920-unofficial-golden": 6,
  "14.1-20171011-unofficial-golden": 50,
  "14.1-20171215_192311-official-golden": 85,
  "14.1-20171215_192311-unofficial-golden": 493,
  "14.1-20180109_002536-unofficial-golden": 119,
  "14.1-20181014-unofficial-golden": 1,
  "14.1-20200817_071197-official-golden-rogerwed": 1,
  "15.1-20181030-unofficial-golden": 5,
  "golden-user 8.1.0 oe632e 3de271a2ce release-keys": 1,
  "njh47f release-keys": 1
}